plugins/gemstallhelper

```diff
@@ -0,0 +1,3 @@
+repository=https://github.com/Pepijn-V/Ardy-Gem-Stall.git
+commit=e640352a2d61a3b82e23c80d7860594876a70ab4
+authors=PepijnV
```

Comment:
This needs to match your actual GitHub handle (i.e. `Pepijn-V`, though it may be lowercase if you want).